aws/apis/compute/v1alpha1/types.go

```diff
@@ -220,6 +209,8 @@ type EKSClusterStatus struct {
 	State string `json:"state,omitempty"`
 	// ClusterName identifier
 	ClusterName string `json:"resourceName,omitempty"`
```

Comment:
Do we want the JSON fields to match the struct fields here?

Reply:
That's a good question, but seems like that hasn't been the case (see `ClusterName string` one line above)

Reply:
I think we'll offer a better experience matching the upstream fields precisely (within the conventions of Go variable naming).  No guessing required for those that know the AWS API, tools, or terminology.

We could take advantage of the field documentation to more elaborately explain that the resource name is the name of the EKS Cluster.  Likewise for `ClusterVersion`.  Are these the only fields that do not conform to their remote names?

There are no embedding conflicts with `ResourceName` and `ResourceVersion` https://github.com/crossplaneio/crossplane/blob/master/apis/core/v1alpha1/resource.go#L143-L146